test/fit_tests/config/global_config.json

```diff
@@ -0,0 +1,83 @@
+{
+  "credentials": {
+    "_comment": "This is the list of credentials for each type of node or service",
+    "hyper": [
+      {
+        "username": "root",
+        "password": "1234567"
+      },
+      {
+        "username": "root",
+        "password": "password"
+      }
+    ],
+    "ora": [
+      {
+        "username": "onrack",
+        "password": "onrack"
+      }
+    ],
+    "bmc": [
+      {
+        "username": "admin",
+        "password": "admin"
+      },
+      {
+        "username": "root",
+        "password": "1234567"
+      }
+    ],
+    "node": [
+      {
+        "username": "root",
+        "password": "1234567"
+      },
+      {
+        "username": "onrack",
+        "password": "onrack"
+      }
+    ],
+    "switch": [
+      {
+        "username": "admin",
+        "password": "1234567"
+      }
+    ],
+    "pdu": [
+      {
+        "username": "admn",
+        "password": "admn"
+      }
+    ]
+  },
+  "snmp":{
+    "community": "onrack"
+  },
+  "repos": {
+    "_comment": "This is the list of repositories for each category",
+    "proxy": "http://web.hwimo.lab.emc.com:3128",
```

Comment:
That is problem , the global_config file contains many internal links. @gpaulos can we create two global_config.json for public github and internal github, Only the one in internal Github have that internal links?